test/wasm2js/left-to-right.2asm.js

```diff
@@ -942,23 +942,23 @@ function asmFunc(imports) {
  }
  
  function $53() {
-  var wasm2js_i32$0 = 0, wasm2js_i32$1 = 0;
+  var wasm2js_i32$0 = 0, wasm2js_i32$2 = 0;
```

Comment:
Do you know why the names changed?

Reply:
This diff was because of the extra temp vars - unneeded ones get optimized out, but the optimizer happened to pick other ones here. Anyhow, this part of the diff goes away in the last commit.